db/config/config.js

```diff
@@ -0,0 +1,24 @@
+const env = process.env.NODE_ENVIRONMENT || 'local'
+
+const test = require('dotenv').config({path: `${__dirname}/./../../.env.${env}`});
```

Comment:
`${__dirname}/./../../.env.${env}` is the extra single `/./` needed?

Reply:
No it is not!